apps/rush-lib/src/logic/ProjectChangeAnalyzer.ts

```diff
@@ -16,7 +16,10 @@ import { BaseProjectShrinkwrapFile } from './base/BaseProjectShrinkwrapFile';
 import { RushConfigurationProject } from '../api/RushConfigurationProject';
 import { RushConstants } from './RushConstants';
 
-export class PackageChangeAnalyzer {
+/**
+ * @beta
+ */
+export class ProjectChangeAnalyzer {
   /**
    * null === we haven't looked
```

Comment:
This is not an intuitive usage of `null`.  Maybe you could replace it with the [UNINITIALIZED](https://github.com/microsoft/rushstack/blob/78aa5aded8051d9f38e8cf0d1513f20707edc54a/apps/rush-lib/src/logic/taskRunner/ProjectBuilder.ts#L69) token from **ProjectBuilder.ts** or just split it into two variables.